### ISSUE TITLE
t2 crash without info

### ISSUE THREAD

**Issue by tristobal** [tessel/t2-cli#730]
Hi there.

I install t2-cli (in Ubuntu 16.04) as follows:
```
sudo npm install -g --unsafe-perm t2-cli
```
And the installation was successfull, but when i execute the t2 command a "crash" is throwed without any details:

```
cristobal@dell $ t2
INFO Crash Reported: http://crash-reporter.tessel.io/
```

Any idea?

**Comment by johnnyman727** [comment]
@tristobal I recommend you check out https://github.com/tessel/t2-cli/issues/727 for some clues on what might be wrong. I do document in that thread how I got it working on Ubuntu 14.04 but I haven't tried with Ubuntu 16.

**Comment by tristobal** [comment]
But, is there a way of obtein more detailed info? Not only "INFO Crash Reported"?

Thanks anyway.

**Comment by wyze** [comment]
I think #696 actually outputs the error to the console. Unfortunately this was landed right after `0.0.19` was released.

**Comment by jimbojw** [comment]
This blocking issue prevents Ubuntu users from troubleshooting their common installation problems. See #714

**Comment by johnnyman727** [comment]
@tristobal as @jimbojw described in #727, this is mostly an issue with the package we use for prompting CLI users to update when a new release is available. After `sudo t2 install-drivers` is run on Linux, it rewrites the permissions of the update config file which makes subsequent attempts to read from it throw an error. Our crash reporter software was originally swallowing errors but that was fixed with #696. The permissions issue was somewhat fixed with #715. If you update your CLI to `0.0.20` or later, this should be fixed.

**Comment by tristobal** [comment]
Solved upgrading the CLI version.

Thanks.
